pkg/reconciler/service/service_test.go

```diff
@@ -1573,7 +1594,7 @@ func TestNew(t *testing.T) {
 	}
 }
 
-func config(name, namespace string, so ServiceOption, co ...ConfigOption) *v1alpha1.Configuration {
+func makeConfiguration(name, namespace string, so ServiceOption, co ...ConfigOption) *v1alpha1.Configuration {
```

Comment:
can we leave this as `config` - I'm looking to swap this reconciler to use v1 APIs and this change would prevent a clean merge/rebase for me

Reply:
I'd rather we just land the migration first, it is more important since it unblocks starting the deprecation clock on the alpha/beta types.